pyne/partisn.py

```diff
@@ -41,9 +41,14 @@
 
 print(HAVE_PYMOAB)
```

Comment:
This isn't from you're PR, but I wonder if there is a reason for including this print statement here?

Reply:
I have no idea why that was there. I can remove it. I don't see it as necessary.